code/modules/awaymissions/corpse.dm

```diff
@@ -84,18 +88,14 @@
 			W.assignment = corpseidjob
 		W.registered_name = M.real_name
 		M.equip_to_slot_or_del(W, slot_wear_id)
-	if(src.coffin == 1)
+	if(coffin == 1)
 		var/obj/structure/closet/coffin/sarcophagus/sarc = locate(/obj/structure/closet/coffin/sarcophagus) in loc
 		if(sarc) M.loc = sarc
 	qdel(src)
 
-
-
 // I'll work on making a list of corpses people request for maps, or that I think will be commonly used. Syndicate operatives for example.
-
-
-
-
+/obj/effect/landmark/corpse/damaged
+	brute_damage = 1000

```

Comment:
Brute damage won't ever gib a mob; this is done to max out its damage all over and give it a ton of damage overlays.